### DISCUSSION TITLE
为什么全局安装失败？？

### DISCUSSION THREAD

**Discussion by llds66** [slidevjs/slidev#1802] [Q&A] [answered]
![image](https://github.com/user-attachments/assets/202fa110-c5dc-4709-8aed-bce31c80c5f4)

**Comment by kermanx** [comment, marked as answer]
查了一下，vue@3.4.36 一小时前才发布，有可能是镜像源还没有同步🤔

**Reply by llds66** [reply]
感谢回复，刚刚试了下，成功了
